resources/bolt_bash_completion.sh

```diff
@@ -6,74 +6,134 @@
 ###
 ### Marc Schoechlin ms-github@256bit.org
 
-_bolt()
```

Comment:
I think you need a rebase since this looks like work from another PR?

Reply:
I think this is just cause it's against main instead of master. Updated